pkg/openSearch/open_search_query/boolQueryBuilder_test.go

```diff
@@ -389,5 +392,7 @@ func TestFilterQueryOperatorOr(t *testing.T) {
 }
 
 func prepareFilterFieldMapping() {
-	filterFieldMapping = map[string]string{"testName": "testName"}
+	SetQuerySettings(QuerySettings{
+		FilterFieldMapping: map[string]string{"testName": "testName"},
+	})
```

Comment:
this initialization of `QuerySettings` will be overwritten by subsequent SetQuerySettings calls in the tests, therefore I suggest to remove the function